LICENSE.md

```diff
@@ -1,7 +1,21 @@
-Copyright 2021 Remix Software Inc.
+MIT License
 
-Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+Copyright (c) Remix Software 2021
```

Comment:
I'm no lawyer but the [RR version](https://raw.githubusercontent.com/remix-run/react-router/main/LICENSE.md) of this has a year range.  Do we need that for Remix or does the year indicate the start of the copyright period?  Seems tedious to have to update the range every year.  If it carries forward automatically we may want to remove the end year in RR?  Probably a @mjackson question though...

Reply:
> or does the year indicate the start of the copyright period

I thought it was like "we copyright this starting the mentioned year"

Reply:
@mjackson says starting year is fine